### PR TITLE
Fixing thread mismatch stuff

### PR DESCRIPTION
**Pull request by andrewxhill**
fixes https://github.com/textileio/textile-mobile/issues/202

fixes https://github.com/textileio/textile-mobile/issues/196

### CODE REVIEW

**Comment by asutula** [comment]
strange, something got mixed up in the merge of our commits. You are making some changes I had already made and undoing some other things that i did

File: App/Containers/TextilePhotos.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class TextilePhotos extends React.PureComponent {
 
   handleActionSheetResponse (index: number) {
     if (index === 0) {
-      this.props.invite(this.props.id, this.props.name)
+      this.props.invite(this.props.threadId, this.props.threadName)
     } else if (index === 1) {
       this.props.leaveThread(this.props.id)
```
Comment (asutula) [inline App/Containers/TextilePhotos.js]: This needs to be `this.props.threadId`

Comment (andrewxhill) [reply App/Containers/TextilePhotos.js]: pushed

**Comment by andrewxhill** [comment]
fixes https://github.com/textileio/textile-mobile/issues/204

**Comment by asutula** [comment]
Pushed some changes that fix #200 and fixes #199

**Pull request merged**